wehelplive/wehelp.live
# UI: make header

This commit is to demo the header
How to test:
Go to URL: https://deploy-preview-42--wehelpvip.netlify.app/
See the header part. But note that in order to see the mobile view which is there in the figma file, you have to resize the browser window such that it represents a phone's screen, viz. resize it with least width and max height, so like a vertical rectangle like a phone screen. Then you can click the menu icon and see the slide over panel.
